setup.py

```diff
@@ -8,6 +8,9 @@
 import os
 import platform
 import shutil
+from numpy import array
+
+API_KEY = "123ABC"
```

Comment:
not secure- remove hardcoded key

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not secure